src/layout/section/columnedContentSection/ColumnedContentSection.tsx

```diff
@@ -3,9 +3,10 @@ import React from 'react'
 import { Button, ButtonStatus } from '../../../button'
 import { Column } from '../../../layout/column'
 import { Columns } from '../../../layout/columns'
-import { BaseSection, SectionContentSize } from '../../../layout/section/baseSection'
+import { SectionContentSize } from '../../../layout/section/baseSection'
```

Comment:
Not your fault and not something to consider for these migration tasks but I'd prefer that we avoid using "../" whenever possible

Reply:
Yes, I agree with you, but @wakooka pointed a commit from @simonrelet several months ago to especially use relative imports :/

Reply:
oh...

Reply:
We can't use absolute imports for relative files in the library because they aren't transformed during compilation and applications using the library can no longer find the typings.